[BUG] Error while processing mail - CELERY

### Description

I have problem with Dev branch with Celery. When Paperless consume email with attacment, error is occure :

 Error while processing mail 76999: consume_file() got an **unexpected keyword argument 'task_name'**

### Steps to reproduce

1. Create mail rule
2. Error is occure

### Webserver logs

```bash
[2022-11-06 00:03:18,951] [DEBUG] [paperless_mail] Rule ***@gmail.com.***: Processing mail *** from ***@mail.com with 1 attachment(s)

[2022-11-06 00:03:18,954] [INFO] [paperless_mail] Rule ***@gmail.com.***: Consuming attachment ***.pdf from mail *** from ***@mail.com

[2022-11-06 00:03:18,955] [ERROR] [paperless_mail] Rule ***@gmail.com.***: Error while processing mail 76999: consume_file() got an unexpected keyword argument 'task_name'

Traceback (most recent call last):

  File "/usr/src/paperless/src/paperless_mail/mail.py", line 318, in handle_mail_rule

    processed_files = self.handle_message(message, rule)

  File "/usr/src/paperless/src/paperless_mail/mail.py", line 417, in handle_message

    consume_file.delay(

  File "/usr/local/lib/python3.9/site-packages/celery/app/task.py", line 425, in delay

    return self.apply_async(args, kwargs)

  File "/usr/local/lib/python3.9/site-packages/celery/app/task.py", line 540, in apply_async

    check_arguments(*(args or ()), **(kwargs or {}))

TypeError: consume_file() got an unexpected keyword argument 'task_name'
```


### Browser logs

_No response_

### Paperless-ngx version

dev

### Host OS

Ubuntu

### Installation method

Docker - official image

### Browser

_No response_

### Configuration changes

_No response_

### Other

_No response_

--- a/src/paperless_mail/mail.py
+++ b/src/paperless_mail/mail.py
@@ -425,7 +425,6 @@ class MailAccountHandler(LoggingMixin):
                     else None,
                     override_document_type_id=doc_type.id if doc_type else None,
                     override_tag_ids=tag_ids,
-                    task_name=att.filename[:100],
                 )
 
                 processed_attachments += 1

